java/org/contikios/cooja/Simulation.java

```diff
@@ -53,13 +51,27 @@
  *
  * @author Fredrik Osterlind
  */
-public class Simulation extends Observable implements Runnable {
+public class Simulation extends Observable {
+
+  /* Internal marker for notifying simulation thread to start running simulation */
```

Comment:
Add a `*` to these comments so IntelliJ will show them when mousing over START/STOP/QUIT.

Reply:
Since it is only for internal use, I avoided this so it would not end up in Javadoc, but I can change this.